docs/api/authentication.rst

```diff
@@ -0,0 +1,136 @@
+.. _authenticating:
+
```

Comment:
This page has to be contrasted with `api/cliquet/authentication.rst`.

IMO we should remove its inclusion and duplicate its content here.